### ISSUE TITLE
Implement Lovell designs and launch

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9656]

**Comment by ContentBrewmaster** [comment]
Added epic template content and assigned CM values.

**Comment by EWashb** [comment]
@jilladams or @kmariepat-cityfriends can this epic be closed now?

**Comment by jilladams** [comment]
@EWashb Yes! Thank you. (I'm in the middle of a project to try and lay out the current state of things for Facilities epics, and hopefully we'll be closing a _bunch_ of old epics, coming up. 🤞 )
